Sources/Property.swift

```diff
@@ -9,7 +9,31 @@ import enum Result.NoError
 ///
 /// Only classes can conform to this protocol, because having a signal
 /// for changes over time implies the origin must have a unique identity.
-public protocol PropertyProtocol: class, BindingSource {
+#if swift(>=4.1)
+	public protocol PropertyProtocol: class, BindingSource {
```

Comment:
Nit: I would avoid this indentation to simplify the diff.

Reply:
yeah... I checked other places to see if we wanted to 'indent' around a #if... and it's not consistent.  I think you indent inside a class/struct definition, but idk.  I DID try and match the style here!